peak/mapper/binding.py

```diff
@@ -142,26 +128,26 @@ def enumerate(self):
 
     def get_enumerate(self,t):
         #custom enumeration
-        if t in self.enumeration_scheme:
+        try:
             return self.enumeration_scheme[t]
-
-        #default scheme
-        if is_adt_type(t):
-            def gen():
-                return t.enumerate()
-            return gen
-        elif issubclass(t,SMTBitVector):
-            def gen():
-                for val in (0,-1):
-                    yield t(val)
-            return gen
-        elif issubclass(t,SMTBit):
-            def gen():
-                for val in (0,1):
-                    yield t(val)
-            return gen
-        else:
-            raise ValueError(str(t))
+        except:
```

Comment:
Should be:
```
try:
  return ...
except KeyError:
  pass
#default scheme
```

Reply:
Fixed

Reply:
Should be `except KeyError:` not raw `except`